IFComp/lib/IFComp/Schema/Result/User.pm

```diff
@@ -46,6 +46,8 @@ __PACKAGE__->table("user");
   is_nullable: 0
   size: 128
 
+User's real name
```

Comment:
These comments popped up because they're in my local MySQL, and I'm surprised they haven't shown up in the schema already. (I have no memory of when I might have created these.)

Not technically on-topic but they snuck into this schema dump and I decided to just leave em.